Build warning in object-packer.cpp

```
[40/320] Building CXX object lib/base/CMakeFiles/base.dir/object-packer.cpp.o
../lib/base/object-packer.cpp:267:9: warning: moving a local object in a return statement prevents copy elision [-Wpessimizing-move]
        return std::move(result);
               ^
../lib/base/object-packer.cpp:267:9: note: remove std::move call here
        return std::move(result);
               ^~~~~~~~~~      ~
1 warning generated.
[57/320] Building CXX object lib/base/CMakeFiles/base.dir/tcpsocket.cpp.o
```

--- a/lib/base/object-packer.cpp
+++ b/lib/base/object-packer.cpp
@@ -262,7 +262,5 @@ String icinga::PackObject(const Value& value)
 	StringBuilder builder;
 	PackAny(value, builder);
 
-	String result;
-	result.insert(result.End(), builder.begin(), builder.end());
-	return std::move(result);
+	return String(builder.begin(), builder.end());
 }

